examples/src/main/java/io/dapr/examples/workflows/DemoWorkflowClient.java

```diff
@@ -45,22 +36,16 @@ public class DemoWorkflowClient {
    * @throws InterruptedException If program has been interrupted.
    */
   public static void main(String[] args) throws InterruptedException, TimeoutException {
-    DurableTaskClient client = new DurableTaskGrpcClientBuilder().build();
-    try (client) {
-      String name = DemoWorkflow.class.getCanonicalName();

```

Comment:
consider keeping this instead of using "ExampleWorkflow" as `DemoWorkflow.class.getCanonicalName()` is a registered workflow (as long as the DemoWorkflowService code is running)